.github/workflows/tests.yml

```diff
@@ -77,15 +74,17 @@ jobs:
           fi
 
           echo "MIN_COVERAGE=$MIN_COVERAGE" >> "$GITHUB_OUTPUT"
-      - name: Run Unit tests
-        run: poetry run pytest -m "not azure and not functional" --junitxml=coverage-junit.xml --cov=. --cov-report xml:coverage.xml --cov-fail-under ${{ steps.coverage-value.outputs.MIN_COVERAGE }}
-      - uses: actions/upload-artifact@v4
```

Comment:
We've lost the publishing of the coverage artifact so we no longer have coverage commenting/fail thresholds. Is there any way we can get the coverage files out of the container so we can still publish the artifact?